src/common/components/select-repository-list/index.js

```diff
@@ -70,7 +70,7 @@ class SelectRepositoryList extends Component {
     const { selectedRepos } = this.props.selectRepositoriesForm;
     if (selectedRepos.length) {
       // TODO: Switch to batched repository action
```

Comment:
This TODO comment is obsolete now, isn't it?  That is, we still need changes inside `createSnaps` but not here.